back/spec/acceptance/initiative_status_changes_spec.rb

```diff
@@ -124,34 +126,63 @@
-        before do
-          SettingsService.new.activate_feature! 'initiative_review'
+      # If the review feature is off, the initiative_status_change with code 'proposed'
+      # is created at the model level when the initiative is published.
+      # Thus, this POST request can only happen when the review feature is on.
+      context 'when the the status change is to the proposed status' do
+        let(:new_initiative) { create(:initiative) }
+        let!(:_initiative_status_change) do
+          create(
+            :initiative_status_change,
+            initiative: new_initiative,
+            initiative_status: @status_review_pending
+          )
+        end
+
+        let(:initiative_id) { new_initiative.id }
+        let(:initiative_status_id) { @status_proposed.id }
+
+        example 'creation results in locked editing' do
+          expect(Initiative.find(initiative_id).editing_locked).to be false
 
-          configuration = AppConfiguration.instance
-          configuration.settings['initiatives']['require_review'] = true
-          configuration.save!
+          do_request
+          assert_status 201
+          expect(Initiative.find(initiative_id).editing_locked).to be true
+        end
+
+        example "creation results in the logging of an Initiative 'proposed' activity" do
+          expect { do_request }
+            .to have_enqueued_job(LogActivityJob)
+            .with(instance_of(Initiative), 'proposed', @user, instance_of(Integer))
+            .exactly(1).times
         end
+      end
 
+      context 'when the the status change is to the changes_requested status' do
         let(:new_initiative) { create(:initiative) }
         let!(:_initiative_status_change) do
           create(
             :initiative_status_change,
-            initiative: new_initiative, initiative_status: create(:initiative_status_review_pending)
+            initiative: new_initiative,
+            initiative_status: @status_changes_requested
           )
         end
 
         let(:initiative_id) { new_initiative.id }
-        let(:initiative_status_id) { @status_proposed.id }
+        let(:initiative_status_id) { @status_review_pending.id }
 
-        example 'Create a status change to the proposed status' do
-          expect(Initiative.review_required?).to be true
+        example 'creation does not result in locked editing' do
           expect(Initiative.find(initiative_id).editing_locked).to be false
 
           do_request
           assert_status 201
+          expect(Initiative.find(initiative_id).editing_locked).to be false
+        end
 
-          expect(Initiative.find(initiative_id).editing_locked).to be true
+        example "creation does not result in the logging of an Initiative 'proposed' activity" do
+          expect { do_request }
+            .not_to have_enqueued_job(LogActivityJob)
+            .with(instance_of(Initiative), 'proposed', anything, anything)
```

Comment:
I would probably combine these two tests into one for the sake of performance. Our BE tests is the slowest CI part atm.

Reply:
Same comment as here: https://github.com/CitizenLabDotCo/citizenlab/pull/5631/#discussion_r1300044826